backend/src/models/users.ts

```diff
@@ -0,0 +1,47 @@
+import { AnyTxtRecord } from 'dns';
+import mongoose from 'mongoose';
+
+interface userInterface{
+    name: String;
```

Comment:
Use `string` not `String`. Check this out for understanding the difference - https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string